test/local.mk

```diff
@@ -79,14 +81,23 @@ $(envtest_crd_dir)/helm.crossplane.io_releases.yaml: $(.envtest_crd_dir)
 $(envtest_crd_dir)/helm.crossplane.io_providerconfigs.yaml: $(.envtest_crd_dir)
 	curl -sSL -o $@ $(provider_helm_download_root)/helm.crossplane.io_providerconfigs.yaml
 
-.envtest_crds: .envtest_crd_dir $(envtest_crd_dir)/helm.crossplane.io_releases.yaml $(envtest_crd_dir)/helm.crossplane.io_providerconfigs.yaml
+$(envtest_crd_dir)/k8up.io_schedules.yaml: $(.envtest_crd_dir)
+	curl -sSL -o $@ $(k8up_download_root)/k8up.io_schedules.yaml
+
+.envtest_crds: .envtest_crd_dir $(envtest_crd_dir)/helm.crossplane.io_releases.yaml $(envtest_crd_dir)/helm.crossplane.io_providerconfigs.yaml  $(envtest_crd_dir)/k8up.io_schedules.yaml
 
 ####
 #### S3 Bucket
 ####
 
-bucket_namespace ?= default
-
 .PHONY: s3-credentials
-s3-credentials: minio-setup ## Copies Minio's S3 Bucket credentials to $(bucket_namespace)
-	kubectl -n minio-system get secret minio-server -o yaml | yq e 'del(.metadata) | .metadata.name = "s3-credentials"' | kubectl -n $(bucket_namespace) apply -f -
+s3-credentials: export KUBECONFIG = $(KIND_KUBECONFIG)
+s3-credentials: install-samples ## Copies Minio's S3 Bucket credentials to $(bucket_namespace)
```

Comment:
Open for a better solution here!

Reply:
Let's not spend too much time on this, consider implementing #74 soon